pkg/apis/volumesnapshot/v1beta1/types.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1alpha1
+package v1beta1
 
 import (
```

Comment:
P1: prevent deleting snapshot while restoring volume from the snapshot.

Reply:
I double checked. We've already had a finalizer on VolumeSnapshot that prevents it from being deleted when restoring volume from the snapshot.  We also have a finalizer on VolumeSnapshotContent to prevent the content from being deleted if it is bound to VolumeSnapshot.